.travis.yml

```diff
@@ -4,8 +4,23 @@ go:
   - 1.8
 
 dist: trusty
-sudo: false
+sudo: required
 
+services:
+  - docker
+cache:
+  directories:
+    - "$HOME/google-cloud-sdk/"
+env:
+  - DOCKER_COMPOSE_VERSION="1.13.0" PATH=$PATH:${HOME}/google-cloud-sdk/bin CLOUDSDK_CORE_DISABLE_PROMPTS=1 PROJECT_NAME=transparent-keyserver
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
```

Comment:
Switched to using pip now which is less hacky (apt doesn't work for docker-compose).